Dockerfile

```diff
@@ -1,5 +1,5 @@
 # sha from https://hub.docker.com/layers/amd64/eclipse-temurin/21-jre-alpine/images/sha256-8179ddc8a6c5ac9af935020628763b9a5a671e0914976715d2b61b21881cefca?context=explore
```

Comment:
Update this to `# sha from https://hub.docker.com/layers/amd64/eclipse-temurin/21.0.6_7-jre-alpine/images/sha256-f184bb601f9e6068dd0a92738764d1ff447ab68c15ddbf8c303c5c29de9a1df8`

Reply:
Thank you, fixed